src/content/a-z-content/index.md.njk

```diff
@@ -0,0 +1,279 @@
+---
+title: HMLR content style guide
+description: If your service is on the service.gov.uk subdomain you must use the GDS Transport font
+section: Content
+layout: layout-pane.njk
+show_page_nav: false
+---
+
+{% from "govuk/components/accordion/macro.njk" import govukAccordion %}
+
+The HMLR style guide is a list of content styles and terms that are specific to HMLR. For example, names of caseworking systems, acronyms and what we call common tasks.
+
+{{ govukAccordion({
```

Comment:
We should ideally use the Nunjucks template so that if the HTML is changed to introduce fixes in the future, we get it for free.

I have started to change it but you may need to continue this @romanidavies

Reply:
Done